app/scripts/modules/google/src/serverGroup/configure/wizard/location/basicSettings.controller.js

```diff
@@ -21,7 +21,7 @@ module.exports = angular.module('spinnaker.google.serverGroup.configure.wizard.b
     function searchImages(q) {
       $scope.command.backingData.filtered.images = [
         {
-          message: '<span class="fa fa-cog fa-spin"></span> Finding results matching "' + q + '"...'
+          message: `<loading-spinner size="'nano'"></loading-spinner> Finding results matching "  ${q}  "...`
```

Comment:
remove space between quotes and ${q}

Reply:
just need to get this one and we're good to merge

Reply:
ping @archana-s

Reply:
My bad. Sorry. Getting it in.